ruby-challenges.rb

```diff
@@ -10,6 +10,22 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# Pseudocode:
+# Define the method odd_or_even
+# Use a conditional statement to check if odd or even
+# Return using string interpolation
+
+def odd_or_even number
+    if number % 2 == 0
+        "#{number} is even"
+    else
+        "#{number} is odd"
+    end
+end
```

Comment:
Nice!  I would recommend looking at some of the built-in methods Ruby has, some of which handle some of this logic for you.